__tests__/raw-capture-api-test.js

```diff
@@ -0,0 +1,512 @@
+      .end(function (err, res) {
+        if (err) return done(err);
+        return done();
+      });
+  });
+
+  it(`Should raise validation error with error code 422 -- attributes should be an array of objects with properties, key and value`, function (done) {
+    const request_object = new RequestObject();
+    request_object.change_property('attributes', [
+      '5000',
+      ...request_object.request_object.attributes,
+    ]);
+    request(server)
+      .post(`/raw-captures`)
+      .send(request_object.request_object)
+      .set('Accept', 'application/json')
+      .expect(422)
+      .end(function (err, res) {
+        if (err) return done(err);
+        return done();
+      });
+  });
+
+  it(`Should raise validation error with error code 422 -- attributes should be an array of objects with properties, key and value`, function (done) {
+    const request_object = new RequestObject();
+    request_object.change_property('attributes', [
+      { text: 'Checking this out' },
+      ...request_object.request_object.attributes,
+    ]);
+    request(server)
+      .post(`/raw-captures`)
+      .send(request_object.request_object)
+      .set('Accept', 'application/json')
+      .expect(422)
+      .end(function (err, res) {
+        if (err) return done(err);
+        return done();
+      });
+  });
+
+  it(`Should raise validation error with error code 422 -- planter_identifier is required `, function (done) {
+    const request_object = new RequestObject();
+    request_object.delete_property('planter_identifier');
+    request(server)
+      .post(`/raw-captures`)
+      .send(request_object.request_object)
+      .set('Accept', 'application/json')
+      .expect(422)
+      .end(function (err, res) {
+        if (err) return done(err);
+        return done();
+      });
+  });
+
+  it(`Should raise validation error with error code 422 -- timestamp is required `, function (done) {
+    const request_object = new RequestObject();
+    request_object.delete_property('timestamp');
+    request(server)
+      .post(`/raw-captures`)
+      .send(request_object.request_object)
+      .set('Accept', 'application/json')
+      .expect(422)
+      .end(function (err, res) {
+        if (err) return done(err);
+        return done();
+      });
+  });
+
+  it(`Should raise validation error with error code 422 -- should be of type unixtimestamp in seconds `, function (done) {
+    const request_object = new RequestObject();
+    request_object.change_property('timestamp', 'ashbfhd');
+    request(server)
+      .post(`/raw-captures`)
+      .send(request_object.request_object)
+      .set('Accept', 'application/json')
+      .expect(422)
+      .end(function (err, res) {
+        if (err) return done(err);
+        return done();
+      });
+  });
+
+  describe('Raw capture should be added sucessfully', () => {
+    before((done) => {
+      // Sql query to create table for testing in the github actions environment
+      knexMainDB.schema
```

Comment:
Also, it is missing the `tree_attributes` table which is also a one that is used to insert data by the legacy logic.